### ISSUE TITLE
Wrong order of WINDOW / QUALIFY clauses

### ISSUE THREAD

**Issue by lukaseder** [jOOQ/jOOQ#11663]
When using both the `WINDOW` and `QUALIFY` clauses, the clauses are generated in the wrong order, e.g.

```sql
select "PUBLIC"."T_BOOK"."ID"
from "PUBLIC"."T_BOOK"
qualify row_number() over "w" > 2
window "w" as (order by "PUBLIC"."T_BOOK"."ID" desc)
order by "PUBLIC"."T_BOOK"."ID"
```

The correct order is:

```sql
select "PUBLIC"."T_BOOK"."ID"
from "PUBLIC"."T_BOOK"
window "w" as (order by "PUBLIC"."T_BOOK"."ID" desc)
qualify row_number() over "w" > 2
order by "PUBLIC"."T_BOOK"."ID"
```

**Comment by lukaseder** [comment]
Commit will be done with https://github.com/jOOQ/jOOQ/issues/5810

**Comment by lukaseder** [comment]
Fixed in jOOQ 3.15.0 and 3.14.9 (#11664) (only H2 is affected in 3.14.9)
